### ISSUE TITLE
Keybindings while editing text

### ISSUE THREAD

**Issue by soegaard** [racket/drracket#282]
Feature request:
  Lines beginning with a semi colon consists of text. Therefore the keybindings ought to 
  work as "text" operations and not "s-exp" operations.

The example that keeps tripping me up: 

  While editing the text in a comment, I hit alt-<left arrow> to go back a word --- and is always surprised when the cursor moves out of the comment to the beginning of the previous s-expression.

In short: I'd love for alt+left and alt+right to become backward-word and forward-word in comment lines.
